sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
Do you need to set `IsClientLibrary` here? You already do that in the .props file next to the project.
Do we need this file at all?